config/locales/en.yml

```diff
@@ -844,6 +845,8 @@ en:
           actions:
             dropdown:
               update: Update
+              delete: Delete
+            delete_confirm: Are you sure you want to remove this metadata field from the sample?
```

Comment:
**question:** Do you think we should at least say the `key` that is going to be deleted?  I can see a user clicking the remove button, seeing the confirmation, and then getting nervous that they might of clicked the wrong one.

Reply:
Agreed, fixed in bb0755d9fe743251a1552eb46bb0e92b16c8441d